crates/bevy_render/src/texture/image.rs

```diff
@@ -85,6 +89,7 @@ impl ImageFormat {
             ImageFormat::Dds => image::ImageFormat::Dds,
             ImageFormat::Farbfeld => image::ImageFormat::Farbfeld,
             ImageFormat::Gif => image::ImageFormat::Gif,
+            ImageFormat::Exr => image::ImageFormat::OpenExr,
```

Comment:
`ImageFormat::Exr` seems to be the only format whose name doesn't exactly match its `image::ImageFormat` counterpart. Could we rename it to `ImageFormat::OpenExr` for full consistency?